src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Backup/Blocks/options.html.twig

```diff
@@ -32,8 +32,8 @@
     <h3 class="card-header">
       <i class="material-icons">settings</i> {{ 'Backup options'|trans }}
     </h3>
-    <div class="card-block row">
-      <div class="card-text">
+    <div class="card-body">
+      <div class="form-wrapper">
```

Comment:
Hi @NeOMakinG 

is there no other way but to have this intermediate div? The previous changes were more intuitive for developers as it reduced the amount of needed code and cleaned useless div and classes.

Now it doesn't clean code it just replaces the former class with a new one meaning developers have even more chances not to correctly integrate this as they wouldn't know about the `form-wrapper` class

Reply:
We could keep card-text, but change the style in order to avoid abusing of bootstrap col style

I changed the name because `card-text` looks like a bootstrap component's part, which is not the case

Reply:
Btw, it does clean the code => card-block doesn't exist, it's not a bootstrap component's part at all

We need this intermediate div because some pages doesn't use it in order to have a full width card content, but maybe we can add a modifier to these pages only, and get rid of this intermediate div !!

Reply:
@jolelievre The original way was wrong joe. 
- Row must always have some columns -> didn't have them here.
- Card-block does not exist anymore, it was only in Alpha.
- Card-text should be only used for texts.

The correct syntax is .card > .card-body. And if we have some kind of forms with limited width, they should be wrapped in something only for that purpose, that's why the form-wrapper. :-)